### PR TITLE
[PAY-1020] Allow gated track in same session as wallet connect

### PR DESCRIPTION
**Pull request by sddioulde**
### Description

Currently, the collectibles state is not properly refreshed when a user connects or disconnects a wallet. This makes it so that they would need to refresh their session before uploading a collectible gated track right after connecting a wallet that has valid nft collections.
This PR fixes that.

This is only the fix for web.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

local web app vs stage

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Comment by nicoback2** [comment]
> It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

@sddioulde you can ignore this, it was commented an error (need to figure out what went wrong)

**Comment by sliptype** [comment]
> > It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype
> 
> @sddioulde you can ignore this, it was commented in error (need to figure out what went wrong)

Is it the common changes?

**Comment by nicoback2** [comment]
> > > It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype
> > 
> > 
> > @sddioulde you can ignore this, it was commented in error (need to figure out what went wrong)
> 
> Is it the common changes?

@sliptype yeah it's the combination of mobile src changes + non mobile changes since the command is
`! git diff --quiet HEAD origin/main -- 'packages/mobile' ':!packages/mobile/src' || circleci-agent step halt`

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sd-connect

File: packages/common/src/store/pages/profile/reducer.ts [hunk 1]
```diff
@@ -227,6 +229,12 @@ const actionsMap = {
   },
   [FETCH_COLLECTIONS_FAILED](state, action) {
     return updateProfile(state, action, { collectionStatus: Status.ERROR })
+  },
+  [REFRESH_WALLET_COLLECTIBLES](state, action) {
```
Comment (piazzatron) [inline packages/common/src/store/pages/profile/reducer.ts:233]: nit: do we even need these here

Comment (sddioulde) [reply packages/common/src/store/pages/profile/reducer.ts:233]: dont think we do. mechanically added and forgot to remove. good eyes

**Review by rickyrombo — CHANGES_REQUESTED** [review]
This was superseded by #3000

**Pull request closed without merging**